src/python/docs/sphinx/make.bat

```diff
@@ -1,7 +1,7 @@
 @ECHO OFF
 
 pushd %~dp0
-set PYTHONINTERPRETER=%~dp0..\..\..\..\dependencies\Python3.6\python.exe
+set PYTHONINTERPRETER=%~dp0..\..\..\..\dependencies\Python7\python.exe
```

Comment:
Should this be `Python3.7`?

Reply:
Fixed.

Reply:
no need for this. docs can be generated by python 3.6

---
In reply to: [272979237](https://github.com/Microsoft/NimbusML/pull/101#discussion_r272979237) [](ancestors = 272979237)